docs/docsite/rst/community/communication.rst

```diff
@@ -17,7 +17,7 @@ All communication and interactions in the Ansible Community are governed by our
 Forum
```

Comment:
For most readers I hope they only need to read  CoC & Forum.
So lets see if we can provide all the relevant context (without been too long) in these sections.